test/cases/mirax-slidedat-bad-image-concat-factor/config.yaml

```diff
@@ -1,5 +1,5 @@
 base: Mirax/CMU-1.zip
-error: ^Key file contains key ('|“)IMAGE_CONCAT_FACTOR('|”) in group ('|“)LAYER_0_LEVEL_2_SECTION('|”) which has (a )?value that cannot be interpreted.$
```

Comment:
How is this change related to the new glib version?

Reply:
The regex matches a glib error string.  We can stop matching variants produced by older glib versions.